spec/whats/configuration_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Whats, ".configuration" do
+  subject(:configuration) { Whats.configuration }
+
+  context "when no configuration is done" do
+    before "clean whats module" do
```

Comment:
As Whats is a module it's scope is global, so other specs can
(and will) setup the configuration before the test, which checks
when there's no configuration. Therefore we need a way to clean the
configs

Reply:
Maybe create a reset method for configs?